JavaAssignments/src/vishal/ConvertTemp.java

```diff
@@ -0,0 +1,15 @@
+package vishal;
+
+public class ConvertTemp {
+	public float convertTemp(float fahTemp) {
+		float cel = (fahTemp - 32) * 5 / 9;
+		return cel;
+	}
+
+	public static void main(String[] args) {
+		ConvertTemp ct = new ConvertTemp();
+		float convertedTemp = ct.convertTemp(12);
```

Comment:
You should use run time parameter  ie. args[0]

Reply:
Done with changes.